<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordView</class>
 <widget class="QFrame" name="RecordView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="sampleLabel">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="sampleLineEdit">
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>48</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveAsBtn">
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>48</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Save As</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveUpBtn">
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>48</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Move Up</string>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hideRowBtn">
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>48</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Hide Row</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hideColumnBtn">
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>48</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Hide Column</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearBtn">
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>48</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
